migrator/data/DataImporter.java

```diff
@@ -361,97 +377,160 @@ private void insertEntity(TypeDB.Transaction transaction, DataProto.Item.Entity
             EntityType type = transaction.concepts().getEntityType(msg.getLabel());
             if (type == null) throw TypeDBException.of(TYPE_NOT_FOUND, msg.getLabel());
             Entity entity = type.create();
-            recordCreated(entity.getIID(), msg.getId());
+            recordMapping(entity.getIID(), msg.getId());
             status.entityCount.incrementAndGet();
         }
     }
 
-    private class CompleteRelations extends ImportWorker {
+    private class Relations extends ImportWorker {
```

Comment:
Relations always import incrementially - if there are any role players imported, we immediately create the relation, then mark it is incomplete if we are missing any role players. The relations will be completed iteratively